app/models/theme_field.rb

```diff
@@ -277,6 +277,82 @@ def target_name
     Theme.targets.invert[target_id].to_s
   end
 
+  class ThemeFileMatcher
+    OPTIONS = %i{name type target}
+    def initialize(regex, canonical:, target:, name:, type:)
+      @allowed_values = {}
+      @allowed_values[:name] = Array(name) if name
```

Comment:
you can use `[name]` instead of `Array(name)` to be more "ruby"-like here. (and for the following two lines too!)

Reply:
I think the behaviour is slightly different. Here I wanted to have the option of passing either a single symbol, or an array of symbols into the function. Later on it should always be an array. Using the `[name]` syntax would end up with a two-dimensional array:
```
[1] pry(main)> test = :hello
=> :hello
[2] pry(main)> [test]
=> [:hello]
[3] pry(main)> test2 = [:hello, :world]
=> [:hello, :world]
[4] pry(main)> [test2]
=> [[:hello, :world]]
```
Using `Array()`:
```
[5] pry(main)> Array(test)
=> [:hello]
[6] pry(main)> Array(test2)
=> [:hello, :world]
```

Reply:
I see. However, then I think the code has a clarity problem. It's not obvious by reading the signature that `name` could be a single symbol or an array of symbols. Maybe something like `names` or `types` would make more sense here.

Reply:
:+1: I've pluralised the three arguments to make it clearer. Thanks